data/json/overmap_specials.json

```diff
@@ -1,1770 +1,1867 @@
-        "city_distance" : [3, 5],
-        "city_sizes" : [1, 12],
-        "occurrences" : [75, 100],
-        "flags" : [ "UNIQUE" ]
-    },{
-        "type" : "overmap_special",
-        "id" : "bandit_cabin",
-        "overmaps" : [
-            { "point":[0,0,0], "overmap": "bandit_cabin"}
-        ],
-        "locations" : [ "forest" ],
-        "city_distance" : [20, -1],
-        "city_sizes" : [4, 12],
-        "occurrences" : [1, 1],
-        "rotate" : false,
-        "flags" : [ "CLASSIC" ]
-    },{
-        "type" : "overmap_special",
-        "id" : "bandit_camp",
-        "overmaps" : [
-            { "point":[0,0,0], "overmap": "bandit_camp_1"},
-            { "point":[1,0,0], "overmap": "bandit_camp_2"},
-            { "point":[0,1,0], "overmap": "bandit_camp_3"},
-            { "point":[1,1,0], "overmap": "bandit_camp_4"}
-        ],
-        "locations" : [ "forest" ],
-        "city_distance" : [20, -1],
-        "city_sizes" : [4, 12],
-        "occurrences" : [1, 1],
-        "rotate" : false,
-        "flags" : [ "CLASSIC" ]
-    },{
-        "type" : "overmap_special",
-        "id" : "sai",
-        "overmaps" : [
-            { "point":[0,0,0], "overmap": "sai" }
-        ],
-        "connections" : [
-            { "point" : [0,-1,0], "terrain" : "road" }
-        ],
-        "locations" : [ "wilderness", "swamp" ],
-        "city_distance" : [-1, 2],
-        "city_sizes" : [2, 14],
-        "occurrences" : [0, 4],
-        "rotate" : false
-    },{
-        "type" : "overmap_special",
-        "id" : "power_station_small",
-        "overmaps" : [
-            { "point":[0,0,0], "overmap": "pwr_sub_s" }
-        ],
-        "connections" : [
-            { "point" : [0,-1,0], "terrain" : "road" }
-        ],
-        "locations" : [ "wilderness", "swamp" ],
-        "city_distance" : [-1, 6],
-        "city_sizes" : [2, 14],
-        "occurrences" : [0, 4],
-        "rotate" : false,
-        "spawns" : { "group" : "GROUP_SMALL_STATION", "population":[4, 10], "radius":[2, 4] }
-    },{
-        "type" : "overmap_special",
-        "id" : "power_station_large",
-        "overmaps" : [
-            { "point":[0,1,0], "overmap": "pwr_large_entrance"},
-            { "point":[-1,1,0], "overmap": "pwr_large_2"},
-            { "point":[-1,0,0], "overmap": "pwr_large_3"},
-            { "point":[0,0,0], "overmap": "pwr_large_4"}
-        ],
-        "connections" : [
-            { "point" : [0,2,0], "terrain" : "road" }
-        ],
-       "locations" : [ "wilderness" ],
-        "city_distance" : [5, 16],
-        "city_sizes" : [4, 14],
-        "occurrences" : [1, 3],
-        "rotate" : false,
-        "spawns" : { "group" : "GROUP_LARGE_STATION", "population":[4, 10], "radius":[1, 2] }
-    },{
-        "type" : "overmap_special",
-        "id" : "warehouse",
-        "overmaps" : [ { "point":[0,0,0], "overmap": "warehouse_north" } ],
-        "connections" : [
-            { "point" : [0
```

Comment:
Looks like you linted the entire file, which made the diff huge.
Linting of this whole file would be useful, but it would need to be a totally separate PR.
For now just manually paste new entries into un-linted version of the file.

Reply:
That was a mistake (thought was already linted previously) and I rolled it back.